thirdparty/jstree/jquery.jstree.js

```diff
@@ -1,5 +1,5 @@
 /*
- * jsTree 1.0-rc3
+ * jsTree 1.0-rc3 (patched)
```

Comment:
Maybe make this a bit more specific like `jsTree custom for SilverStripe (1.0-rc3 + minor bugfixes)`